### ISSUE TITLE
CLIC or CLINT?

### ISSUE THREAD

**Issue by lbmeng** [starfive-tech/beaglev_doc#2]
In https://github.com/starfive-tech/beaglev_doc/raw/main/JH7100%20Data%20Sheet%20V01.01.04-EN%20(4-21-2021).pdf,

on page#12, it says the following for "SiFive U74 Dual-Core":
> CLIC for timer and software interrupts

and on page#13, it says the following for "SiFive E24 Core":
> CLIC with support for up to 127 interrupts with 16 priority levels

Are they correct? Should it be **CLINT**?

**Comment by pdp7** [comment]
Yes, I think you are correct that it should be CLINT.  I believe SiFive uses Core-Local Interruptor (CLINT) _(see page 31 in [SiFive U7 manual](https://github.com/starfive-tech/beaglev_doc/blob/main/vic_u7_manual_with_creativecommons.pdf))_ as their implementation of the CLIC.

@davidlt @tekkamanninja is that correct?

**Comment by MichaelZhuxx** [comment]
For U74 Dual-Core, use CLINT and PLIC, please refer to [https://github.com/starfive-tech/beaglev_doc/blob/main/vic_u7_manual_with_creativecommons.pdf](url)

For E24 Core，use CLIC [https://www.sifive.com/cores/e24](url)

**Comment by lbmeng** [comment]
Thanks for the clarification!

**Comment by pdp7** [comment]
I am leaving open until there is new PDF

**Comment by whuhujin** [comment]
U7 core supports CLINT + PLIC , and E24 supports CLIC.
